lib/circuitry/configuration.rb

```diff
@@ -8,6 +8,7 @@ class Configuration
     attribute :subscriber_queue_name, String
     attribute :subscriber_dead_letter_queue_name, String
     attribute :publisher_topic_names, Array[String]
+    attribute :subscriber_topic_names, Array[String]
```

Comment:
Should we add a default of `[]` for both of these just to ensure they're always treated as arrays?